src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -74,6 +81,16 @@
     ShibServiceBean shibService;
     @EJB
     AuthTestDataServiceBean authTestDataService;
+    @EJB
+    UserServiceBean userService;
+
+    // Make the session available
+    @Inject
+    DataverseSession session;    
+
+    public static final String listUsersPartialAPIPath = "list-users";
+    public static final String listUsersFullAPIPath = "/api/v1/admin/" + listUsersPartialAPIPath;
```

Comment:
Is there a way to re-use the url* string for the API--let's talk in person
(* or have a url has an object)